integration-tests/keycloak-authorization/src/main/java/io/quarkus/it/keycloak/ProtectedResource.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.it.keycloak;
+
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.security.auth.AuthPermission;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.keycloak.representations.idm.authorization.Permission;
+
+import io.quarkus.security.identity.SecurityIdentity;
+
+@Path("/api/permission")
+public class ProtectedResource {
+
+    @Inject
+    SecurityIdentity identity;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<Permission> permissions() {
+        if (identity.checkPermissionBlocking(new AuthPermission("Permission Resource"))) {
```

Comment:
@pedroigor I'll add a follow up issue for us to come up with Quarkus permission class so that the users can write a portable code with or without KC policy enforcer, not a prob for this PR 
CC @stuartwdouglas